README.md

```diff
@@ -1 +1,209 @@
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [📝 License ](#-license-)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Space Travellers's Hub <a name="about-project"></a>
+
+**Space traveler's hub** is a single-page website where users can reserve rockets and join space missions.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+- React.js
+- Redux, Redux Toolkit
+- React-Bootstrap
+- Jest
+- Node.js
+- HTML/CSS/SCSS
+
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+> Describe between 1-3 key features of the application.
+
+- **You can reserve rocket**
+- **You can joing missions**
+- **You can view reservation on my profile page**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+> Add a link to your deployed project.
+
+ - [Live Demo Link]() 
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+> Describe how a new developer could make use of your project.
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+<!--
+Example command:
+
+```sh
+ gem install rails
+```
+ -->
+
+### Setup
+Clone this repository to your desired folder:
+
+```sh
+  cd my-folder
+  git clone https://github.com/Itswali/React-Group.git .
+```
+
+
+### INSTALL
+
+Before installing the app, make sure to install node.js in your machine. In order to check if node is installed run the following command:
+
+```sh
+  node -v
+```
+
+Install this project with:
+
+```sh
+  npm install
+```
+
+### Usage
+
+To run the project in a development server, execute the following command:
+
+```sh
+  npm start
+```
+
+### Testing
+
+To run the tests in a development server, execute the following command:
+
+```sh
+  npm test
+```
+
+To to build for the production, execute the following command:
+
+```sh
+  npm run build
+```
+
+### Deployment
+
+You can deploy this project using [Github Pages](https://docs.github.com/en/pages/getting-started-with-github-pages/creating-a-github-pages-site)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+> Mention all of the collaborators of this project.
+
+👤 **Wali-Muhammad**
+
+- GitHub: [@Itswali](https://github.com/Itswali)
+- Twitter: [@Wali-Muhammad](https://twitter.com/WaliMuh94818599)
+- LinkedIn: [@Wali Muhammad](https://www.linkedin.com/in/wali-muhammad-666040244/)
+
+👤 **Hein Thant**
+
+- GitHub: [@githubhandle](https://github.com/indiecodermm)
+- Twitter: [@twitterhandle](https://twitter.com/hthant_oo)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/hthantoo)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+> Describe 1 - 3 features you will add to the project.
```

Comment:
- Please, let's remove this placeholder text as this are there to help you customize your readme by updating the contents to suit your project needs. Please, do this to make your readme more professional.